src/cmd/manage_nsfs.js

```diff
@@ -108,15 +108,25 @@ const accounts_dir_name = '/accounts';
 const access_keys_dir_name = '/access_keys';
 
 async function check_and_create_config_dirs(config_root) {
-    const bucket_dir_stat = await fs_utils.file_exists(config_root);
-    if (!bucket_dir_stat) {
-        await fs_utils.create_path(config_root);
-        await fs_utils.create_path(config_root + buckets_dir_name);
-        await fs_utils.create_path(config_root + accounts_dir_name);
-        await fs_utils.create_path(config_root + access_keys_dir_name);
-        return true;
+    const pre_req_paths = [
+        config_root,
+        path.join(config_root, buckets_dir_name),
+        path.join(config_root, accounts_dir_name),
+        path.join(config_root, access_keys_dir_name)
+    ];
+    for (const p of pre_req_paths) {
```

Comment:
Let's make the `p` variable name `path`

Reply:
path can't be re defined, it's a const at the beginning of the file